content/services/weiterentwicklung-apps-produkte/weiterentwicklung-apps-produkte.md

```diff
@@ -0,0 +1,48 @@
+---
+templateKey: 'service-page'
+slug: /weiterentwicklung-apps-produkte
+title: Weiterentwicklung von Apps und Produkten
+image: hero.png
+description: Wir führen die Entwicklung von Apps und Software-Produkten nahtlos weiter.
+subtitle:
+  text: Wir führen die Entwicklung von Apps und Software-Produkten nahtlos weiter.
+callToAction: Sie möchten die Weiterentwicklung Ihrer Software-Produkte mit Apptiva besprechen?
+---
+
+## Wenn es nicht so läuft, wie es sollte
+
+Im Zeitalter der Digitalisierung gibt es sehr viele Hersteller von Software-Produkten. Die Zufriedenheit der Kunden und die Qualität der Arbeit hat allerdings nicht für alle Hersteller oberste Priorität. So kann es durchaus vorkommen, dass Kunden sich gezwungen sehen, sich nach anderen Herstellern umzusehen. Glücklicherweise ist noch keiner unserer Kunden aus den genannten Gründen "abgesprungen". Wir durften aber schon einige Apps und Produkte übernehmen, welche ursprünglich nicht von uns entwickelt wurden.
+
+Folgende Ausgangslagen haben wir schon angetroffen:
+
+- Die Software ist sehr fehleranfällig. Bei zusätzlichen Funktionen kommen immer auch neue Fehler mit.
+- Der Hersteller ist sehr unzuverlässig. Termine werden nicht eingehalten.
+- Supportanfragen werden immer erst mit grosser Verspätung behandelt.
+
+Selbstverständlich kann es auch vorkommen, dass ein Software-Hersteller eine App oder ein Produkt nicht mehr weiterentwickelt; sei es aus strategischen oder wirtschaftlichen Gründen.
+
+Es gibt verschiedenste Varianten, diese Probleme anzupacken. Eine davon ist: man sucht sich einen neuen Software-Hersteller.
+
+## Unterstützung von Apptiva
```

Comment:
Für SEO ist es immer gut wenn die Keywords auch in den Titeln sind. Würde deshalb hier schreiben: Weiterentwicklung durch Apptiva